mw-webapp/src/pages/usersPage/UsersPage.tsx

```diff
@@ -0,0 +1,6 @@
+/**
+ * Page all users
```

Comment:
if page named UserPage, why in description you have PageAllUsers? What the difference? It's not a literature project :) try to avoid synonims

Reply:
done)